specs/supported-color-schemes-meta.html

```diff
@@ -4,31 +4,31 @@
 <link rel=stylesheet href=https://resources.whatwg.org/standard-shared-with-dev.css>
 <body>
      <dt><dfn id=meta-supported-color-schemes><code>supported-color-schemes</code></dfn><dd>
-               <p>The value defines the default supported color schemes for the document and allow the UA to
-               know what backdrop color to draw early on to avoid white flash before rendering a dark themed
+               <p>The value defines the default supported color schemes for the document and allows the UA to
+               load a scheme-specific variant of the default UA stylesheet to avoid white flash before rendering a dark themed
```

Comment:
This modification based on what I learned through https://github.com/w3c/csswg-drafts/issues/3299#issuecomment-479380882.